lib/api/env-impl.ts

```diff
@@ -12,6 +12,13 @@ class ENV implements Environment {
   public get debug(): boolean {
     return env.debug;
   }
+
+  public set packMode(value: boolean) {
```

Comment:
'pack' instead of 'packMode' for consistency?

Reply:
Updated.

---
In reply to: [586187863](https://github.com/microsoft/onnxjs/pull/266#discussion_r586187863) [](ancestors = 586187863)